### DISCUSSION TITLE
how to remove pdfarranger ?

### DISCUSSION THREAD

**Discussion by EliRhamba** [pdfarranger/pdfarranger#980] [Q&A] [answered]
Hello everyone
I'm a complete begginer in Linux, so maybe my question will sound ridiculous to you, I apologize in advance. I'm not sure it's the right place to ask this, but I have not found the information elsewhere. 
I would like to remove pdfarranger, I had installed it with this command : 

> flatpak install flathub com.github.jeromerobert.pdfarranger

I have tried to remove it with this command, but it didn't work : 

> sudo apt-get --purge remove flathub com.github.jeromerobert.pdfarranger

I have also tried to remove it with this command : 

> sudo apt-get --purge remove python3-pip python3-wheel python3-gi python3-gi-cairo \
    gir1.2-gtk-3.0 gir1.2-poppler-0.18 gir1.2-handy-1 python3-setuptools

But pdfarranger is still there. Could you help me please ?

**Comment by jeromerobert** [comment, marked as answer]
As you installed with flatpak see, https://docs.flatpak.org/en/latest/using-flatpak.html#remove-an-application
